calculator/src/App.js

```diff
@@ -0,0 +1,12 @@
+import { Component } from 'react';
+import Calculator from './component/Calculator';
+import './App.css';
+export default class App extends Component {
+  render() {
+    return (
+      <div id="app">
+        <Calculator></Calculator>
```

Comment:
[children 이 없다면 혼자서 닫아버릴 수도 있답니다.](https://reactjs.org/docs/jsx-in-depth.html#:~:text=You%20can%20also%20use%20the%20self%2Dclosing%20form%20of%20the%20tag%20if%20there%20are%20no%20children.%20So%3A) 
```suggestion
        <Calculator />
```

Reply:
앗 children이 없는데  self-closing을 사용하지 않았군요. 수정하겠습니다